src/index.html

```diff
@@ -1,16 +1,49 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="get"
+      data-qa="keypress"
+    >
+      <label>
+        <img
+          class="search-icon"
+          src="./images/Search.svg"
+          alt="search-icon"
+        >
+        <input
+            data-qa="big"
+            placeholder="Try &quot;Los Angeles&quot;"
+            type="search"
+            class="input"
+        >
+      </label>
+    </form>
+    <form action="/" method="get">
+      <label>
+        <img
+          class="search-icon icon-small"
+          src="./images/Search.svg"
+          alt="search-icon-small"
+        >
+        <input
+          data-qa="small"
+          placeholder="Try &quot;Los Angeles&quot;"
+          type="search"
+          class="small"
```

Comment:
```suggestion
        <input
          data-qa="small"
          placeholder="Try &quot;Los Angeles&quot;"
          type="search"
          class="input input-small"
```
checklist p.2, read carefully